tests/std/tests/P0226R1_math_special_functions/test.cpp

```diff
@@ -0,0 +1,542 @@
+        assert(isclose(legendrel(3, 0.25L), -0.3359375L));
+    }
+    {
+        assert(isclose(legendre(4, 0.25), 0.15771484375));
+        assert(isclose(legendre(4, 0.25f), 0.15771484375f));
+        assert(isclose(legendre(4, 0.25L), 0.15771484375L));
+        assert(isclose(legendref(4, 0.25f), 0.15771484375f));
+        assert(isclose(legendrel(4, 0.25L), 0.15771484375L));
+    }
+}
+
+void test_laguerre() {
+    // laguerre(x, y) = LaguerreP[x, y]
+    {
+        assert(isnan(laguerre(3, nand_v)));
+        assert(isnan(laguerre(3, nanf_v)));
+        assert(isnan(laguerre(3, nanl_v)));
+        assert(isnan(laguerref(3, nanf_v)));
+        assert(isnan(laguerrel(3, nanl_v)));
+    }
+    {
+        assert(isclose(laguerre(1, 0.5), 0.5));
+        assert(isclose(laguerre(1, 0.5f), 0.5f));
+        assert(isclose(laguerre(1, 0.5L), 0.5L));
+        assert(isclose(laguerref(1, 0.5f), 0.5f));
+        assert(isclose(laguerrel(1, 0.5L), 0.5L));
+    }
+    {
+        assert(isclose(laguerre(2, 0.5), 0.125));
+        assert(isclose(laguerre(2, 0.5f), 0.125f));
+        assert(isclose(laguerre(2, 0.5L), 0.125L));
+        assert(isclose(laguerref(2, 0.5f), 0.125f));
+        assert(isclose(laguerrel(2, 0.5L), 0.125L));
+    }
+}
+
+void test_riemann_zeta() {
+    // riemann_zeta(x) = Zeta[x]
+    {
+        assert(isnan(riemann_zeta(nand_v)));
+        assert(isnan(riemann_zeta(nanf_v)));
+        assert(isnan(riemann_zeta(nanl_v)));
+        assert(isnan(riemann_zetaf(nanf_v)));
+        assert(isnan(riemann_zetal(nanl_v)));
+    }
+    {
+        assert(isclose(riemann_zeta(-1.0), -0.0833'333'333'333'333'333));
+        assert(isclose(riemann_zeta(-1.0f), -0.0833'333'333'333'333'333f));
+        assert(isclose(riemann_zeta(-1.0L), -0.0833'333'333'333'333'333L));
+        assert(isclose(riemann_zetaf(-1.0f), -0.0833'333'333'333'333'333f));
+        assert(isclose(riemann_zetal(-1.0L), -0.0833'333'333'333'333'333L));
+    }
+    {
+        assert(isclose(riemann_zeta(0.0), -0.5));
+        assert(isclose(riemann_zeta(0.0f), -0.5f));
+        assert(isclose(riemann_zeta(0.0L), -0.5L));
+        assert(isclose(riemann_zetaf(0.0f), -0.5f));
+        assert(isclose(riemann_zetal(0.0L), -0.5L));
+    }
+}
+
+void test_sph_bessel() {
+    // sph_bessel(x, y) = sqrt[pi/(2 * x)] BesselJ[y + 1/2, x]
+    {
+        assert(isnan(sph_bessel(1, nand_v)));
+        assert(isnan(sph_bessel(1, nanf_v)));
+        assert(isnan(sph_bessel(1, nanl_v)));
+        assert(isnan(sph_besself(1, nanf_v)));
+        assert(isnan(sph_bessell(1, nanl_v)));
+    }
+    {
+        assert(isclose(sph_bessel(1, 1.2345), 0.352'106'432'958'031'885));
+        assert(isclose(sph_bessel(1, 1.2345f), 0.352'106'432'958'031'885f));
+        assert(isclose(sph_bessel(1, 1.2345L), 0.352'106'432'958'031'885L));
+        assert(isclose(sph_bessel(1, 1.2345f), 0.352'106'432'958'031'885f));
+        assert(isclose(sph_bessel(1, 1.2345L), 0.352'106'432'958'031'885L));

```

Comment:
The last two lines are repeated, instead of testing the `f` and `l` suffixed functions.